pkg/plugin/local/local_test.go

```diff
@@ -195,11 +200,14 @@ func TestLocalDataSource_DestroyTrustZone(t *testing.T) {
 				assert.Len(t, lds.config.TrustZones, 1)
 				// nolint:staticcheck
 				assert.Len(t, lds.config.TrustZones[0].Federations, 0)
-				// Check that the trust zone removal was persisted.
+				// Check that the attestation policy binding was removed too.
+				assert.Len(t, lds.config.ApBindings, 1) // Check that the trust zone removal was persisted.
```

Comment:
s/trust zone/binding/

Reply:
It's correct. The whitespace broke tho, should have been left on the next line. Will be fixed with the federations refactoring.